docs/site/Deploying-to-IBM-Cloud.md

```diff
@@ -120,11 +147,12 @@ const datasourceDb = require('./datasources/db.datasource.json');
 const cfenv = require('cfenv');
 const appEnv = cfenv.getAppEnv();
 
-export async function main(options?: ApplicationConfig) {
+export async function main(options: ApplicationConfig = {}) {
```

Comment:
Hmm why do we want to make `options` a required parameter here?

Reply:
that's good point. That's what I got after generating the app, but I just updated to the latest @loopback/cli and seems like the original line is correct. Will change it back.